README.rst

```diff
@@ -0,0 +1,730 @@
+Porter
+======
+
+Overview
+--------
+NuCypher Porter can be described as the *“Infura for NuCypher”*. Porter is a web-based service that performs
+nucypher-based protocol operations on behalf of applications.
```

Comment:
Just a thought to sprinkle in somewhere:

*This enables applications to behave like a "light client" by delegating network intensive operations like peering and node discovery to porter*

Reply:
👍